scripts/setup-k8s-master.sh.in

```diff
@@ -70,6 +84,7 @@ fi
 
 # Initialize master node
 kubeadm init --config /tmp/kubeadm.yaml
+kubeadm token create --ttl 0 {{ClusterToken}}
```

Comment:
It should be possible to configure the token using `MasterConfiguration.BootstrapTokens` rather than having to call out as a separate step.

Reply:
If that is the case, we'll want to file and track the issue against kubeadm. No issues encoding workarounds in the quickstart, but just need to make sure we don't forget to bubble the issues up for tracking where they belong :)

Reply:
Will do!

Reply:
Seems the old configuration I tried had a typo, user error! Changed this to use the configuration instead.